tasks/fix-cat2.yml

```diff
@@ -1423,12 +1562,18 @@
       - RHEL-07-040520
 
 - name: "MEDIUM | RHEL-07-040560 | PATCH | An X Windows display manager must not be installed unless approved."
-  command: "true"
+  yum:
+      name:
+          - "@X Windows System"
```

Comment:
Ok. I'm going to leave it for the time being and make an issue so I can work w/ you to do this correctly. 

As discussed earlier, would probably be best to register the vars from a group request instead of hard coding. Just need to figure out how to approach it. Could use your insight on it. :+1: